.github/workflows/auto-update.yml

```diff
@@ -0,0 +1,14 @@
+# This workflow automatically updates PR branches with latest changes on target branch.
+# See: https://github.com/marketplace/actions/auto-update
+name: autoupdate
+on:
+  # This will trigger on all pushes to all branches.
+  push: {}
+jobs:
+  autoupdate:
+    name: autoupdate
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: docker://chinthakagodawita/autoupdate-action:v1
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
I have used this action.
Based on my experience, we better set some options.
- Default github token can't work to bot opened PR, see [details](https://github.com/marketplace/actions/auto-update#limitations). A solution is using a personal key to replace it.
- Not all PRs need to update. Use a label to trigger this workflow or other conditions. Need to set `PR_FILTER`, `PR_READY_STATE` and `PR_LABELS`.

Reply:
Great comments, @Zeroto521! I took a look at the Auto Update [docs](https://github.com/marketplace/actions/auto-update) and I decided on the following configuration:

- Still do all PRs, b/c I don't think we are at the level of sophistication yet where we leverage labels on PRs, and
- Only do them on PRs that are "ready for review", as I've seen examples of us do PRs in a draft state first.